technical-resources/protocol-overview/smart-contract-addresses.md

```diff
@@ -4,140 +4,156 @@
-| LoanManagerFactory       | [`0x1551717AE4FdCB65ed028F7fB7abA39908f6A7A6`](https://etherscan.io/address/0x1551717AE4FdCB65ed028F7fB7abA39908f6A7A6) | [`pool-v2            @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0)      |
-| MapleLoanFactory         | [`0x36a7350309B2Eb30F3B908aB0154851B5ED81db0`](https://etherscan.io/address/0x36a7350309B2Eb30F3B908aB0154851B5ED81db0) | [`loan               @ v4.0.0`](https://github.com/maple-labs/loan/releases/tag/v4.0.0)         |
-| PoolManagerFactory       | [`0xE463cD473EcC1d1A4ecF20b62624D84DD20a8339`](https://etherscan.io/address/0xE463cD473EcC1d1A4ecF20b62624D84DD20a8339) | [`pool-v2            @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0)      |
-| WithdrawalManagerFactory | [`0xb9e25B584dc4a7C9d47aEF577f111fBE5705773B`](https://etherscan.io/address/0xb9e25B584dc4a7C9d47aEF577f111fBE5705773B) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0)      |
+| FixedTermLoanFactory | [`0x36a7350309B2Eb30F3B908aB0154851B5ED81db0`](https://etherscan.io/address/0x36a7350309B2Eb30F3B908aB0154851B5ED81db0) | [`fixed-term-loan @ v2.0.0`](https://github.com/maple-labs/fixed-term-loan/releases/tag/v2.0.0) |
+| FixedTermLoanManagerFactory | [`0x1551717AE4FdCB65ed028F7fB7abA39908f6A7A6`](https://etherscan.io/address/0x1551717AE4FdCB65ed028F7fB7abA39908f6A7A6) | [`fixed-term-loan-manager @ v1.0.0`](https://github.com/maple-labs/fixed-term-loan-manager/releases/tag/v1.0.0) |
+| LiquidatorFactory | [`0xa2091116649b070D2a27Fc5C85c9820302114c63`](https://etherscan.io/address/0xa2091116649b070D2a27Fc5C85c9820302114c63) | [`liquidations @ v2.0.0`](https://github.com/maple-labs/liquidations/releases/tag/v2.0.0) |
+| OpenTermLoanFactory | [`0x6Fad515Fc046DD17166453A79725f50b917b7cF6`](https://etherscan.io/address/0x6Fad515Fc046DD17166453A79725f50b917b7cF6) | [`open-term-loan @ v1.0.1`](https://github.com/maple-labs/open-term-loan/releases/tag/v1.0.1) |
+| OpenTermLoanManagerFactory | [`0x90b14505221a24039A2D11Ad5862339db97Cc160`](https://etherscan.io/address/0x90b14505221a24039A2D11Ad5862339db97Cc160) | [`open-term-loan-manager @ v1.0.0`](https://github.com/maple-labs/open-term-loan-manager/releases/tag/v1.0.0) |
+| PoolManagerFactory | [`0xE463cD473EcC1d1A4ecF20b62624D84DD20a8339`](https://etherscan.io/address/0xE463cD473EcC1d1A4ecF20b62624D84DD20a8339) | [`pool-v2 @ v1.0.0`](https://github.com/maple-labs/pool-v2/releases/tag/v1.0.0) |
+| WithdrawalManagerFactory | [`0xb9e25B584dc4a7C9d47aEF577f111fBE5705773B`](https://etherscan.io/address/0xb9e25B584dc4a7C9d47aEF577f111fBE5705773B) | [`withdrawal-manager @ v1.0.0`](https://github.com/maple-labs/withdrawal-manager/releases/tag/v1.0.0) |
 
 ## Active Pool Contracts
```

Comment:
we're missing cash mgmt and Aqru pools, if you double check against https://github.com/maple-labs/protocol-registry/blob/main/v2-protocol.md